### PR TITLE
Get basic structure for bizlegfoss profiles and add my first profile.

### PR DESCRIPTION
**Pull request by CCoffie**
Created a basic structure for the bizlegfoss profiles.

### CODE REVIEW

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+## Rationale
+Wordpress is a gigantic thing. It powers (at least) 23% of the web.
+(http://w3techs.com/technologies/details/cm-wordpress/all/all) We wanted to
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: Could you add text to this link? Even something like \[w3techs](http://w3techs.com/technologies/details/cm-wordpress/all/all) would be fine. No one likes raw links.

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  - Plinky - Writing prompts
+  - Code Garage - Backups and security scanning
+  - Simperium and Simplenote - Note taking and data synchronization
+  - Poster - Wordpress mobile app
+  - Lean Domain Search - Domain name search
+  - Cloudup - File-sharing service
+  - Longreads - Sharing service for long form posts
+  - Scroll Kit - Visual HTML editor
+  - Parka - Security scanning
+  - Code For the People - Wordpress development firm
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: Could you add links for these? At least for those that are still around.

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+(http://wptavern.com/a-look-back-at-16-automattic-acquisitions-since-2007)
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: A title on this link would also be nice.

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+
+8. Has the company made any investments in other companies? If yes, which ones.
+  - No they have not. They normally just purchase the companies.
+
+9. Number of Employees?
+  - 306 (http://automattic.com/about/)
+
+10. Where is HQ?
+  - 132 Hawthorne St, San Francisco, CA 94107 (https://www.crunchbase.com/organization/automattic)
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: Title

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+14. Does your organization file any annual reports? Please include links to any
+relevant documents (i.e. 990, Annual Report, Year in Review, etc...)
+  - Bi-annual transparency reports: http://transparency.automattic.com/
+
+## Communications
+1. Does your subject participate in social media? If yes, please list a URL for
+each account, and reach within that community. (i.e. Twitter: @RedHatNews -
+61.9K Followers.)
+  - Twitter: @wordpress - 442K followers
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: Links to twitter accounts would be nice.

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+2. What communication channels does your subject use to reach their public?
+Briefly describe and include a URL for each.
+  - Users of the SaaS product can get notifications and other info on their
+  landing pages. Users of their own installations can get similar notifications.
+  - The Twitter handle mentioned above appears to be more or less a release log.
+
+3. Does your subject organize or participate in any conferences? If so, list
+them here, and provide links to any relevant sessions, keynotes, or content.
+  - Wordcamp - first organized by Matt Mullenweg, but numerous spinoffs have
+  been established. http://central.wordcamp.org/about/
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: Title.

File: chapters/ecosystems/BizLegFoss/Wordpress.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+  - There’s a conference aggregator named Lanyrd that has a Wordpress tag:
+  http://lanyrd.com/topics/wordpress/
```
Comment (ColdenCullen) [inline chapters/ecosystems/BizLegFoss/Wordpress.md]: Title.

**Comment by CCoffie** [comment]
I updated my profile with the suggested changes and a few more changes.

**Comment by ColdenCullen** [comment]
LGTM :+1:

**Comment by msoucy** [comment]
The directory structure was updated for the profiles, please update

**Comment by CCoffie** [comment]
I think I updated everything to work with the new directory structure. Sorry for the mess...

**Comment by qwertos** [comment]
Looks good. Merged. :smile:

**Pull request merged**